tests/EdgeToEdge/Routes/NewDonationRouteTest.php

```diff
@@ -35,31 +45,59 @@ public function paymentInputProvider(): array {
 					'zahlweise' => 'BEZ',
 					'periode' => '0'
 				],
-				'valid'
+				[
+					'validity' => 'valid',
+					'formattedAmount' => '100,00',
+					'isCustomAmount' => false
+				]
 			],
 			[
 				[
 					'amountGiven' => '123.45',
 					'zahlweise' => 'PPL',
 					'periode' => 6
 				],
-				'valid'
+				[
```

Comment:
These diffs confuse me because they look like 17e5b8e & 17e5b8e which are already in the main line. Does this need a rebase?